var/spack/repos/builtin/packages/boost/package.py

```diff
@@ -44,15 +45,35 @@ class Boost(Package):
     version('1.34.1', '2d938467e8a448a2c9763e0a9f8ca7e5')
     version('1.34.0', 'ed5b9291ffad776f8757a916e1726ad0')
 
+    libs = ['chrono', 
+        'date_time', 
+        'filesystem', 
+        'iostreams', 
+        'random', 
+        'regex', 
+        'serialization', 
+        'signals', 
+        'system', 
+        'thread', 
+        'wave', 
+        'mpi', 
```

Comment:
And Boost can build against multiple MPI implementations at the same time, but I assume that projects which consume Boost.MPI don't really handle this case properly, so probably less of a problem.